diff --git a/tests/unit_tests/services_tests/test_ocr_service.py b/tests/unit_tests/services_tests/test_ocr_service.py
index aee3487e..c4710759 100644
--- a/tests/unit_tests/services_tests/test_ocr_service.py
+++ b/tests/unit_tests/services_tests/test_ocr_service.py
@@ -7,7 +7,6 @@ from mealie.services.ocr.pytesseract import OcrService
 ocr_service = OcrService()
 
 
-@pytest.mark.skip("Tesseract is not reliable between environments")
 def test_image_to_string():
     with open(Path("tests/data/images/test-ocr.png"), "rb") as image:
         result = ocr_service.image_to_string(image)
@@ -15,7 +14,6 @@ def test_image_to_string():
             assert result == expected_result.read()
 
 
-@pytest.mark.skip("Tesseract is not reliable between environments")
 def test_image_to_tsv():
     with open(Path("tests/data/images/test-ocr.png"), "rb") as image:
         result = ocr_service.image_to_tsv(image.read())
